src/docker-in-docker/devcontainer-feature.json

```diff
@@ -23,6 +23,7 @@
         "dockerDashComposeVersion": {
             "type": "string",
             "enum": [
+                "none",
```

Comment:
As we are allowing a new value for this Feature option, can we update the minor version in here?
https://github.com/devcontainers/features/blob/89ac66a01556a2133da4cfd6a0608e80c0e42d2c/src/docker-in-docker/devcontainer-feature.json#L3